hazelcast/src/main/java/com/hazelcast/client/impl/protocol/parameters/ExceptionResultParameters.java

```diff
@@ -33,77 +34,88 @@
      * ClientMessageType of this message
      */
     public static final int TYPE = ResponseMessageConst.EXCEPTION;
+    public int errorCode;
     public String className;
-    public String causeClassName;
     public String message;
-    public String stacktrace;
+    public StackTraceElement[] stackTrace;
+    public int causeErrorCode;
```

Comment:
This is necessary for our tests. We have tests that is testing if cause type correct. I added this to not lose that behaviour.